Orange/widgets/data/owconcatenate.py

```diff
@@ -291,12 +295,42 @@ def send_report(self):
         self.report_items(items)
 
     def merge_domains(self, domains):
+        variables = set(chain.from_iterable(
+            [d.variables + d.metas for d in domains]))
+
+        feature_attrs = defaultdict(list)
+        for var in variables:
+            for domain in domains:
+                if var not in domain:
+                    continue
+                for key, value in domain[var].attributes.items():
+                    feature_attrs[var].append((key, value))
+
         def fix_names(part):
             for i, attr, name in zip(count(), part, name_iter):
                 if attr.name != name:
                     part[i] = attr.renamed(name)
                     self.Warning.renamed_variables()
 
+        def fix_attrs(part):
+            fixed = []
+            for attr in part:
+                attrs = feature_attrs[attr]
+                if len(attrs) > 0:
+                    attr = attr.copy()
+                    attr.attributes = dict(attrs)
+                    # find duplicated keys with different values - can't use
+                    # set because dict is not hashable
```

Comment:
Could you please rephrase this comment, e.g.

"(...) because values are not necessarily hashable"